### PR TITLE
use different endpoints for readiness and liveness

### PR DESCRIPTION
**Pull request by shivasishdas**
-Proto changes https://github.com/razorpay/metro-proto/pull/21/files

### CODE REVIEW

**Review by sunnyagg — APPROVED** [review]

File: internal/health/server.go [hunk 1]
```diff
@@ -7,27 +7,49 @@ import (
...
+	}
+
+	logger.Ctx(ctx).Debugw("metro readiness check", "status", "Healthy")
+	return &metrov1.StatusCheckResponse{
+		ServingStatus: metrov1.StatusCheckResponse_SERVING_STATUS_SERVING,
+	}, nil
+}
+
+// LivenessCheck checks responsiveness of all the dependant resources that the service is using
+func (h *Server) LivenessCheck(ctx context.Context, _ *emptypb.Empty) (*metrov1.StatusCheckResponse, error) {
```
Comment (sunnyagg) [inline internal/health/server.go:32]: by default all grpc apis are being tracked under opentracing, we can remove liveliness prove from there as k8s checks that every sec and adds to lot of volume

Comment (shivasishdas) [reply internal/health/server.go:32]: ah yes! let me filter these out.

Comment (shivasishdas) [reply internal/health/server.go:32]: anyways will make this change in a different PR. Need to figure out the middleware interceptor.

Comment (sunnyagg) [reply internal/health/server.go:32]: Sure, Works for now

Comment (shivasishdas) [reply internal/health/server.go:32]: Tracking here https://github.com/razorpay/metro/issues/183

**Pull request merged**